translations/pt-BR/README.md

```diff
@@ -1,83 +1,85 @@
 
-Experimente este template utilizando codespaces seguindo esses passos:
+## 🧐 Usando o Codespaces
 
-1. Crie um repositório a partir deste template. Use este [link para gerar um repositório](https://github.com/microsoft/codespaces-project-template-py/generate). Você pode tornar o repositório privado ou público, até você.
-1. Navegue até a página principal do repositório recém-criado.
-1. No nome do repositório, use o menu dropdwon em code e, na guia Codespaces, selecione "Create Codespace on main".
-   ![Create Codespace](https://docs.github.com/assets/cb-138303/images/help/codespaces/new-codespace-button.png)
-1. Aguarde enquanto Github inicializa o codespace:
-   ![Creating Codespace](https://github.com/microsoft/codespaces-teaching-template-py/raw/main/images/Codespace_build.png)
+Experimente este modelo de repositório usando o Codespaces seguindo estes passos:
 
-### Inspecione seu ambiente do Codespaces
+1. Crie um repositório a partir deste modelo. Use este [link de criação do repositório](https://github.com/microsoft/codespaces-project-template-py/generate). Você pode tornar o repositório privado ou público, conforme sua preferência.
+2. Antes de criar o codespace, habilite o GitHub Copilot para a sua conta! Se não estiver habilitado, dê uma olhada em [Faça alterações usando o Copilot](#make-changes-using-copilot).
+3. Acesse a página principal do repositório recém-criado.
+4. Abaixo do nome do repositório, use o menu suspenso Code e, na guia Codespaces, selecione "Criar Codespace em main".
+   ![Criar Codespace](https://docs.github.com/assets/cb-138303/images/help/codespaces/new-codespace-button.png)
+5. Aguarde enquanto o GitHub inicializa o codespace:
+   ![Criando Codespace](https://github.com/microsoft/codespaces-teaching-template-py/raw/main/images/Codespace_build.png)
 
-O que você tem neste momento é um ambiente pré-configurado onde todos os tempos de execução e bibliotecas que você precisa já estão instalados - uma experiência de configuração zero.
+### Inspecionando o ambiente do Codespaces
 
-## Executando o Aplicativo
+Neste ponto, você tem um ambiente pré-configurado no qual todos os tempos de execução e bibliotecas de que você precisa já estão instalados - uma experiência de configuração zero.
 
-Este aplicativo Python está usando o FastAPI, uma poderosa framework  Web que auto-documenta seus pontos finais de API. A API tem apenas um ponto final que gera uma sequência pseudoaleatória única que pode ser usada como um token.
+## Executando o aplicativo
 
+Este aplicativo Python está usando o FastAPI, um poderoso framework da web que documenta automaticamente seus endpoints de API. A API tem apenas um endpoint que gera uma sequência pseudoaleatória única que pode ser usada como um token.
 
```

Comment:
Por favor, poderia adicionar um details para explicar o que é um ponto de extremidade? Com base [nesta página de learn](https://learn.microsoft.com/pt-br/azure/machine-learning/concept-endpoints?view=azureml-api-2#endpoints-and-deployments), podemos utilizar o texto:

Um ponto de extremidade é uma URL estável e durável que pode ser usada para solicitar ou invocar o modelo, fornecer as entradas necessárias e obter as saídas de volta. Um ponto de extremidade fornece:

* Uma URL estável e durável (como **coloque o link da API de tempo indicada acima no tutorial**).
* Um mecanismo de autenticação e autorização.
Uma implantação é um conjunto de recursos necessários para hospedar o modelo que executa a inferência real. Um único ponto de extremidade pode conter várias implantações que podem hospedar ativos independentes e consumir recursos diferentes com base no que os ativos reais exigem. Os pontos de extremidade têm um mecanismo de roteamento que pode rotear a solicitação gerada para os clientes para implantações específicas no ponto de extremidade.

Para funcionar corretamente, cada ponto de extremidade precisa ter pelo menos uma implantação, ou seja, estar publicado e acessível.